.github/workflows/release.yaml

```diff
@@ -17,6 +17,21 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      # See https://github.com/helm/chart-releaser-action/issues/6
+      # For now, install the same version of Tiller as the chart-testing Helm
+      # client version: v2.15.2.
+      - name: Install Helm
+        run: |
+          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+          chmod 700 get_helm.sh
+          DESIRED_VERSION=v2.15.2 ./get_helm.sh
```

Comment:
v2.16.1?

Reply:
What's the env for? Doesn't seem to be used.

Reply:
Perhaps the minor version doesn't matter anymore. Initially I did this for compatibility with our current helm/chart-testing-action helm version, but we could remove that here.

The `DESIRED_VERSION` is from the Helm 2 get script: https://github.com/helm/helm/blob/02ad2b118731d012ea248e3ede800aa28fa71301/scripts/get#L79-L94

Reply:
Removed that, so it should just install the most recent version of Helm 2 now